net.sf.eclipsecs.core/build.properties

```diff
@@ -7,6 +7,8 @@ bin.includes = .,\
                plugin.xml,\
                schema/,\
                core.properties,\
-               lib/dom4j-2.1.3.jar
+               lib/dom4j-2.1.3.jar, \
+               lib/metadata-gen-1.0-SNAPSHOT.jar, \
```

Comment:
for early POC it is ok.

but you do not build it in CI. Please update CI(Travis) to build this jar and place it in required location (rather then keep it some jar)
if you need PR with metafiles to be merged to project - let me know. Regenerate metafiles and send me link with PR.